docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,100 @@
+1. Operator location: ***malhar-library***
+2. Available since: ***3.5.0***
+3. Operator state: ***Evolving***
+3. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcPollInputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.html)***
+
+### AbstractJdbcPollInputOperator
+This is the abstract implementation that serves as base class for polling messages from jdbc store. This class doesn’t have any ports.
+
+![AbstractJdbcPollInputOperator.png](images/jdbcinput/operatorsClassDiagram.png)
+
+## Properties, Attributes and Ports
+### <a name="props"></a>Properties of JDBCPoller
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *store* | JDBC Store for connection | [JDBCStore](#JDBCStore) | Yes | N/A |
+| *tableName* | table name to be scanned | String | Yes | N/A |
+| *columnsExpression* | Comma separated list of columns to select from the given table. | String | Yes | N/A |
+| *key* | Primary key column name | String | Yes | N/A |
+| *partitionCount* | Static partitions counts | int | No | 1 |
+| *whereCondition* | Where condition for jdbc query | String | No | N/A |
+| *fetchSize* | The number of rows to fetch by PreparedStatement | int | No | 20000 |
+| *pollInterval* | Interval in milliseconds to poll the DB | int | No | 10000 |
+| *queueCapacity* | Capacity of queue which holds DB data before emiting | int | No | 4096 |
+| *batchSize* | Number of elements  to emit in a bacth | int | No | 2000 |
+
+
+#### <a name="JDBCStore"></a>Properties of JDBC Store (BackendStore)
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *databaseDriver* |JDBC Driver class for connection to JDBC Store. This driver should be there in classpath | String | Yes | N/A |
+| *databaseUrl* | Database url of the form jdbc:subprotocol:subname | String | Yes | N/A |
+| *connectionProps* | Comma seperated connection properties e.g. user:xyz,password:ijk | String | Yes | N/A | File format. | String | Yes | N/A |
+
+## Abstract Methods
+void emitTuple(T tuple): Abstract method that emits tuple extracted from jdbc store.
+T getTuple(ResultSet result): Abstract method to get formatted tuple.
+
+## Partitioning of JDBC Poller
+Only static partitioning is supported for JDBC Poller Input Operator. Configure parameter "partitionCount" to define number of static non-polling partitions. The additional partition will be created to poll newly added records.
+
+```xml
+  <property>
+    <name>dt.operator.{OperatorName}.prop.partitionCount</name>
+    <value>4</value>
+  </property>
+```
+
+where {OperatorName} is the name of the POJOEnricher operator.
+Above lines will partition JdbcPoller statically 4 times. Above value can be changed accordingly to change the number of static partitions.
```

Comment:
```This will create 5 operator instances in all. Four of these will fetch the data which is currently in the table. We call these static partitions. An additional partition will be created which will read any newly added data. We call such a partition as a polling partition, as it `polls` for newly added data.
Changing the value of the property will create corresponding number of static partitions.```

Also describe static and polling partitions above.